hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report.py

```diff
@@ -66,6 +74,7 @@ def _select_sub1(self):
     def _from_sub1(self):
         return """
             hr_attendance ha
+            LEFT JOIN hr_employee AS he ON ha.id = he.id
```

Comment:
Isn't this already in https://github.com/OCA/hr/pull/961/files#diff-cf0797000a4e7329aafb13b0890ce2b5d09834ded662f2db6f898a580835f3fcR104 ?

Reply:
It's necessary to work fine sql and view.

Reply:
Then this is under-performant doing 2 times the same join. We shouldn't add this I'm afraid.